ReserveSpot.Services.Api/Controllers/HomeController.cs

```diff
@@ -0,0 +1,18 @@
+﻿// ------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// ------------------------------------------------
+
+using Microsoft.AspNetCore.Mvc;
+using RESTFulSense.Controllers;
+
+namespace ReserveSpot.Services.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
```

Comment:
Just for sanity purpose, place the [ApiController] first.

Reply:
Updated the Api Controller first